src/styles/_login.scss

```diff
@@ -0,0 +1,82 @@
+@use 'main';
+
+.bookbix-logo {
+  font-family: 'MuseoModerno', sans-serif !important;
+  color: white;
+  position: absolute;
```

Comment:
For a responsive design absolute position is rarely use I think this can change to relavation and fix some of padding to a proper design on Figma